test/components/kubernetes/namespaces_labels_test.go

```diff
@@ -46,6 +46,10 @@ func TestAllNamespacesHaveNameLabels(t *testing.T) {
 			continue
 		}
 
+		if name == "istio-system" {
```

Comment:
Couldn't we add the name label on the `istio-system` Namespace object in the chart? But maybe we could do it in the future as a post-render hook..

Reply:
Why should we bother about the namespaces not managed by us(lokomotive)?

Reply:
IMO they are managed by us, as the code suggests:
```yaml
# XXX: Lokomotive specific config
apiVersion: v1
kind: Namespace
metadata:
  name: {{.Values.istioNamespace}}
```

So If user wants to create a NetworkPolicy against this namespace, they would need the label.

Reply:
Ah right, sorry my bad.